### PR TITLE
Intra-org shipments between bases of the same org

### PR DESCRIPTION
**Pull request by fhenrich33**
firefox_CMKmjabMOM

### CODE REVIEW

**Comment by fhenrich33** [comment]
[2.0 FE: Intra-org shipments between bases of the same org](https://trello.com/c/2rl8WblK/1550-20-fe-intra-org-shipments-between-bases-of-the-same-org)

File: front/src/views/Transfers/CreateShipment/CreateShipmentView.tsx [hunk 1]
```diff
@@ -109,6 +109,7 @@ function CreateShipmentView() {
   const currentBase = allAcceptedTransferAgreements?.data?.base;
   const currentOrganisationLabel = `${currentBase?.organisation?.name} - ${currentBase?.name}`;
   const currentOrganisationId = globalPreferences.organisation?.id;
+  const currentOrganisationBases = globalPreferences.availableBases;
```
Comment (HaGuesto) [inline front/src/views/Transfers/CreateShipment/CreateShipmentView.tsx:112]: available bases only shows the other bases in your organisation if you are a Headof Ops user. We will need to query the bases to create shipments with differently. @pylipp any suggestion how best to query all bases in your organisations even the ones you don not have access to?
Maybe:
```
organisation(id:2) {
      id
      name
      bases {
        id
        name
      }
    }
    ```

File: front/src/views/Transfers/CreateShipment/CreateShipmentView.tsx [hunk 2]
```diff
@@ -136,6 +137,7 @@ function CreateShipmentView() {
           name: agreement.targetOrganisation.name,
           bases: agreement.targetBases,
           agreementId: agreement.id,
+          agreementComment: agreement.comment,
```
Comment (HaGuesto) [inline front/src/views/Transfers/CreateShipment/CreateShipmentView.tsx:140]: pls. add the same line in l. 132

File: front/src/views/Transfers/CreateShipment/CreateShipmentView.tsx (hunk 1 above)
Comment (pylipp) [reply front/src/views/Transfers/CreateShipment/CreateShipmentView.tsx:112]: @HaGuesto that's exactly the way to go.

**Review by HaGuesto — APPROVED** [review]
## code review passed
just on comment in the code

## functional review passed
please hide intra-org tab if ther is no option available for intra-org shipment

File: front/src/views/Transfers/CreateShipment/CreateShipmentView.tsx (hunk 1 above)
Comment (HaGuesto) [reply front/src/views/Transfers/CreateShipment/CreateShipmentView.tsx:112]: @pylipp I have only the name and id of other bases is available through this query. It is the same query used to create transferAgreements, right?

**Comment by aerinsol** [comment]
@HaGuesto @fhenrich33 
Some requested UI changes:
1. The other tab should only show the name of the organization, NOT the name of the base. Here the name of the org is "BoxCare", where Thessaloniki is the current base and CANNOT be selected as one of the options in the dropdown (i.e., you can't send a transfer to yourself). 
2. I'd like to keep the format of the original Figma design, ie.., <semibold>`baseName`</semibold> `orgName`

**Pull request merged**